packages/hydrooj/src/model/contest.ts

```
@@ -1047,6 +1107,37 @@ export const statusText = (tdoc: Tdoc, tsdoc?: any) => (
+    try {
+        _problemList = JSON.parse(text);
+    } catch (e) {
+        throw new ValidationError('problems');
+    }
+    const problems = [] as ContestProblem[];
+    const score = {} as Record<number, number>;
+    for (const p of _problemList) {
+        if (!p.pid || typeof p.pid !== 'number') throw new ValidationError('problems');
+        if (!p.label || typeof p.label !== 'string') throw new ValidationError('problems');
+        if (typeof p.title === 'string' && p.title.length === 0) delete p.title;
+        if (typeof p.score === 'number') {
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)